org.lflang/src/org/lflang/federated/generator/FedGenerator.java

```diff
@@ -330,7 +330,8 @@ public TargetConfig getTargetConfig() {
     try {
       compileThreadPool.awaitTermination(Long.MAX_VALUE, TimeUnit.NANOSECONDS);
     } catch (Exception e) {
-      context.getErrorReporter().reportError("Failure during code generation.");
+      context.getErrorReporter().reportError("Failure during code generation: " + e.getMessage());
```

Comment:
This is actually not the exception triggered by the example in #1785 (it is the other one). I'm thinking we should do the same there?

Reply:
Oh, never mind; line 170 is not in response to an exception but an error reported in one of the code generators.